test/unit/errors/core-loader-error-tests.js

```diff
@@ -0,0 +1,12 @@
+import sinon from 'sinon';
+import { chunkLoadErrorHandler } from 'api/core-loader';
+
+describe('core-loader errors', function () {
+    it('exports a chunkLoadErrorHandler function which throws a JWError', function () {
+        const handler = chunkLoadErrorHandler(105);
+        expect(handler).to.be.a('function');
+        // The handler itself is a function, which returns a function which throws
+        expect(chunkLoadErrorHandler).to.not.throw();
+        expect(handler).to.throw();
```

Comment:
this is great. Could we check that it returns an error and that the code is 105 ?

Reply:
No. It's very hard to make `require.ensure` fail via network conditions in a unit test; webpack injects a script tag into the DOM instead of using `xhr`, so we can't mock it.